tests/fixtures/modules/import/error-strict.src.js

```diff
@@ -0,0 +1,5 @@
+import house from "house";
```

Comment:
Oh actually yeah it *is* currently needed because of expects all tests in `modules/` fail with modules off.

```
  1) modules with error-delete should throw an error for import declarations when sourceType is not `module`:
     AssertionError: expected [Function] to throw an error
```